Marketing Q2 Goals (#8108)
PostHog/posthog.com: contents/teams/marketing/objectives.mdx

**Contributors:** Joe working with Sales & CS team

**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/193)

**We'll know we're successful when:** We're effectively processing deals of all sizes and up/cross-selling users into new features. CS team feel they have everything they need to close 90% of current deals. 

### Become default analytics tool for new AI startups (Lior)

**Rationale:** Our strategy is to get in first and AI startups are generally engineering-led, so strongly within our ICP. A significant portion of recent YC batches are AI startups and they have specific needs we need to cater to.

**Contributors:** Lior

**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/190)

**We'll know we're successful when:** We have 50 customers using the Langfuse integration and strong usage among AI startups in upcoming YC batches.

### Finish alternatives guides (Andy)

**Rationale:** We know these improve visibility and drive signups. We have a long list of guides we can ship and a proven formula we can repeat.

> 
> **Owner:** Andy

Author: ```suggestion
```

> **Contributors:** Andy, Ian, Lior, James

Author: ```suggestion
**Contributors:** Andy, Ian, Lior
```

**What we'll ship:** [See issue](https://github.com/PostHog/posthog.com/issues/7517#issuecomment-2015715230)

**We'll know we're successful when:** We've shipped priority guides.

### Scale newsletter (James)

**Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.

**Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 

**What we'll ship: (WIP)** [See issue](https://github.com/PostHog/meta/issues/192)
- Transition to 1 new issue per week
- Improve content promotion on Twitter and LinkedIn
- Trade way more recommendations with other newsletters
- Get at least 1 new (or old) post to go viral
- Improve visibility on the website
- Do more issues that use our network
- Land a big interview
- Continue newsletter sponsorships

**We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.

### Level up branding (Lottie)

> **Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.

Author: ```suggestion
**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.

```

**Contributors:** Lottie working with Website & Docs

**What we'll ship:** [See Lottie's issue](https://github.com/PostHog/meta/issues/191)

**We'll know we're successful when:** People continue to talk about our brand and sharing the awesome new things we ship.

### Hiring plan for marketing team (Charles)

**Rationale:** Demand on team is rapidly expanding as we launch new products, but this needs to be tempered with hiring correct roles + not over hiring.

**Contributors:** Charles

**What we'll ship:** 12-month hiring plan + budget

**We'll know we're successful when:** We've hired cracked new marketer(s).

## Side quests

- [Figure out if we should invest further in framework tutorial videos](https://github.com/PostHog/posthog.com/issues/7908) (Lior)

> - Ship awesome technical blogs, such as reqritingh our pipeline in Rust, Mr Blobby, HogQL insight rewrites. (Ian)

Reviewer: ```suggestion
- Ship awesome technical blogs, such as rewriting our pipeline in Rust, Mr Blobby, HogQL insight rewrites. (Ian)
```

Increase awareness of PostHog within our ICP and ensure PostHog is a brand that sparks conversation and joy.

## Projects

### Docs that spark joy (Andy)

**Rationale:** Some of our documentation has fallen behind the product and there are pockets that fall well short of our current standards. This is a reputational risk if left unresolved.

**Contributors:** Andy, Ian, Lior + product teams (hopefully)

**What we'll ship: (WIP)**
- Best practice and use case guides for:
  - Product analytics
  - Session replay
  - Feature flags
  - A/B testing
  - Data warehouse (if/when ready)
- Much better HogQL docs
- Web analytics docs (if/when ready)
- Update or remove any old/low quality docs
- Optimize docs index pages for desire paths / FAQs

**We'll know we're successful when:** Feedback from community / on social that our docs are great, and feedback from product teams.

### Drive word of mouth through improving support (Joe)

> **Rationale:** Support processes need optimizing to ensure all users have a great support experience. Good support = better word of mouth growth.

Author: ```suggestion
**Rationale:** Support processes need optimizing to ensure all users have a great support experience. Good support = better word-of-mouth growth.
```

**Contributors:** Joe working with Sales & CS teams

**What we'll ship:** LINK TO ISSUE HERE

**We'll know we're successful when:** Support are hitting their SLAs and we're consistently getting great feedback from users about our support.

### Supporting CS in their sales-led activities (Joe)

**Rationale:** TBD

**Contributors:** Joe working with Sales & CS teams

**What we'll ship:** TBD

> **We'll know we're successful when:** TBD

Author: @joethreepwood I wasn't sure what to put here. Could you flesh this out?

### Become default analytics tool for new AI startups (Lior)

**Rationale:** Our strategy is to get in first and AI startups are generally engineering-led, so strongly within our ICP. A significant portion of recent YC batches are AI startups and they have specific needs we need to cater to.

**Contributors:** Lior

> **What we'll ship:**
> - LINK TO LIOR'S ISSUE HERE

Author: ```suggestion
**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/190)
```

**Contributors:** Andy, Ian, Lior, James

**What we'll ship:**
- LINK TO ISSUE

**We'll know we're successful when:** We've shipped core alternatives guides for A/B testing, feature flags, and web analytics alternatives.

### Scale newsletter (James)

**Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.

**Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 

**What we'll ship: (WIP)**
- Transition to 1 new issue per week
- Improve content promotion on Twitter and LinkedIn
- Trade way more recommendations with other newsletters
- Get at least new (or old) post to go viral
- Improve visibility on the website
- Do more issues that use our network
- Land a big interview
- Continue newsletter sponsorships

**We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.

### Level up branding (Lottie)

**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.

> 
> **Owner:** Lottie

Author: ```suggestion
```

Increase awareness of PostHog within our ICP and ensure PostHog is a brand that sparks conversation and joy.

## Projects

- [Docs that spark joy](#docs-that-spark-joy-andy) (Andy)
- [Drive word of mouth through improving support](#drive-word-of-mouth-through-improving-support-joe) (Joe)
- [Supporting CS in their sales-led activities](#supporting-cs-in-their-sales-led-activities-joe) (Joe)
- [Become default analytics tool for new AI startups](#become-default-analytics-tool-for-new-ai-startups-lior) (Lior)
- [Finish alternatives guides](#finish-alternatives-guides-andy) (Andy)
- [Scale newsletter](#scale-newsletter-james) (James)
- [Level up branding](#level-up-branding-lottie) (Lottie)
- [Ship awesome technical blogs](#ship-awesome-technical-blogs-ian) (Ian)
- [Hiring plan for marketing team](#hiring-plan-for-marketing-team-charles) (Charles)

### Docs that spark joy (Andy)

**Rationale:** Some of our documentation has fallen behind the product and there are pockets that fall well short of our current standards. This is a reputational risk if left unresolved.

**Contributors:** Andy, Ian, Lior + product teams (hopefully)

**What we'll ship: (WIP)**

> - Best practice and use case guides for:
>   - Product analytics
>   - Session replay
>   - Feature flags
>   - A/B testing
>   - Data warehouse (if/when ready)
> - Much better HogQL docs
> - Web analytics docs (if/when ready)
> - Update or remove any old/low quality docs
> - Optimize docs index pages for desire paths / FAQs

Reviewer: It's important to be specific here.

- We already have best practice guides for [product analytics](https://posthog.com/product-engineers/5-ways-to-improve-analytics-data) and [feature flags](https://posthog.com/product-engineers/feature-flag-best-practices).
- How should we deal with adhoc requests? How do we avoid context switching too much whenever another team is launching something or needs a docs update?
- Should we review the SDKs, changelog to ensure that all the most important updates have been documented?

```suggestion
- Best practice guides for:
  - Session replay
  - A/B testing
  - Surveys
- Much better HogQL docs
- Updates to support pipeline team with their goals
- Continue expanding web analytics and data warehouse docs as they roll out
- Update or remove any old/low quality docs
- Optimize docs index pages for desire paths / FAQs
```

Author: Agreed. This was very much WIP based on what we talked about in the meeting.

Can table this to discuss in sprint meeting today ahead of a proper mega issue. Like you say, some of this stuff exists but isn't very discoverable, or maybe needs updating etc.

All your suggestions sound sensible to me.

> **We'll know we're successful when:** Feedback from community / on social that our docs are great, and feedback from product teams.

Reviewer 2: Could we use the docs helpful survey to give this a hard metric? Feels like we should, since we have that function, we have existing data to judge against, and we can use that info to target further changes.

I know we don't _need_ a hard metric for things always, but this is somewhere we can definitely have one and use it to ensure we're making an impact.

> ### Drive word of mouth through improving support (Joe)

Reviewer 2: ```suggestion
### Drive word of mouth by improving support (Joe)
```

> **Rationale:** Support processes need optimizing to ensure all users have a great support experience. Good support = better word-of-mouth growth.

Reviewer 2: ```suggestion
**Rationale:** Support processes need optimizing to ensure all users have a great support experience. Good support sparks joy = better word-of-mouth growth.
```

> **Contributors:** Joe working with Sales & CS teams

Reviewer 2: ```suggestion
**Contributors:** Joe working with Sales & CS team
```

> **What we'll ship:** LINK TO ISSUE HERE

Reviewer 2: ```suggestion
**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/186)
```

> **We'll know we're successful when:** Support are hitting their SLAs and we're consistently getting great feedback from users about our support.

Reviewer 2: ```suggestion
**We'll know we're successful when:** We're hitting our SLAs and our CSAT has either improved or remained stable. CSAT is currently 96% satisfaction. 
```

> ### Supporting CS in their sales-led activities (Joe)

Reviewer 2: ```suggestion
### Support CS in their sales-led activities (Joe)
```

> **Rationale:** TBD

Reviewer 2: ```suggestion
**Rationale:** Sales & CS is changing how it works. There's an opportunity to support Sales in getting larger deals, and to improve processes around how incoming deals work. Deals = Growth. 
```

> **Contributors:** Joe working with Sales & CS teams

Reviewer 2: ```suggestion
**Contributors:** Joe working with Sales & CS team
```

> **What we'll ship:** TBD

Reviewer 2: ```suggestion
**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/193)
```

> **We'll know we're successful when:** TBD

Reviewer 2: ```suggestion
**We'll know we're successful when:** We're effectively processing deals of all sizes and up/cross-selling users into new features. CS team feel they have everything they need to close 90% of current deals. 
```

**Rationale:** Our strategy is to get in first and AI startups are generally engineering-led, so strongly within our ICP. A significant portion of recent YC batches are AI startups and they have specific needs we need to cater to.

**Contributors:** Lior

**What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/190)

**We'll know we're successful when:** We have 50 customers using the Langfuse integration and strong usage among AI startups in upcoming YC batches.

### Finish alternatives guides (Andy)

**Rationale:** We know these improve visibility and drive signups. We have a long list of guides we can ship and a proven formula we can repeat.

**Owner:** Andy
**Contributors:** Andy, Ian, Lior, James

**What we'll ship:**
- LINK TO ISSUE

**We'll know we're successful when:** We've shipped core alternatives guides for A/B testing, feature flags, and web analytics alternatives.

### Scale newsletter (James)

**Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.

**Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 

> **What we'll ship: (WIP)**

Reviewer 2: ```suggestion
**What we'll ship: (WIP)** [See issue](https://github.com/PostHog/meta/issues/192)
```

- Transition to 1 new issue per week
- Improve content promotion on Twitter and LinkedIn
- Trade way more recommendations with other newsletters
- Get at least 1 new (or old) post to go viral
- Improve visibility on the website
- Do more issues that use our network
- Land a big interview
- Continue newsletter sponsorships

> **We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.

Author: @jtemperton discussed this on Friday. I was thinking something like reaching 2,000 new subs over 30 days by end of the quarter (currently around 1,000) – i.e. doubling the growth. Maybe this isn't ambitious enough?

### Level up branding (Lottie)

**Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.
**Contributors:** Lottie working with Website & Docs

**What we'll ship:** [See Lottie's issue](https://github.com/PostHog/meta/issues/191)

**We'll know we're successful when:** People continue to talk about our brand and sharing the awesome new things we ship.

> ### Ship awesome technical blogs (Ian)

Reviewer: I feel like this is kind of sidequesty if we want to put it there, especially because I will be the only one working on it. I can own docs updates if that is the case.

Author: That sounds great. I didn't feel totally comfortable owning docs as a project, so this makes sense to me.

**Rationale:** We don't share our engineering achievements frequently enough and they're opportunity to raise awareness with engineers. Also useful as a recruiting tool to attract cracked engineers.

**Contributors:** Ian

**What we'll ship:**
  - [Rewriting our billion event pipeline in Rust](https://github.com/PostHog/posthog.com/issues/7893)
  - Mr Blobby
  - HogQL insight rewrites

**We'll know we're successful when:** We ship some great articles and/or one of them goes viral on Hacker News.

### Hiring plan for marketing team (Charles)

**Rationale:** Demand on team is rapidly expanding as we launch new products, but this needs to be tempered with hiring correct roles + not over hiring.

**Contributors:** Charles

**What we'll ship:** 12-month hiring plan + budget

**We'll know we're successful when:** We've hired cracked new marketer(s).

## Side quests

- [Figure out if we should invest further in framework tutorial videos](https://github.com/PostHog/posthog.com/issues/7908) (Lior)

## Metrics we care about

- New org signups
- Newsletter signups
- Organic SEO visits

> See [marketing dashboard](https://us.posthog.com/project/2/dashboard/51661) (internal only).

Reviewer: ```suggestion
<PrivateLink url="https://us.posthog.com/project/2/dashboard/51661">Marketing dashboard</PrivateLink>
```
